app/src/sharedTest/java/org/oppia/android/app/topic/TopicActivityTest.kt

```diff
@@ -87,17 +87,18 @@ class TopicActivityTest {
   }
 
   @Test
-  fun testTopicActivity_activityLabel_isDefinedSuccessfully() {
+  fun testTopicActivity_hasCorrectActivityLabel() {
     activityTestRule.launchActivity(
       createTopicActivityIntent(
         internalProfileId,
         FRACTIONS_TOPIC_ID
       )
     )
     val title = activityTestRule.activity.title
-    assertThat(title).isEqualTo(
-      context.getString(R.string.topic_page)
-    )
+
+    /* Verify that the activity label is correct as a proxy to verify
+       TalkBack will announce the correct string when it's read out. */
```

Comment:
```suggestion
    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
    // correct string when it's read out.
```

While a block comment is perfectly fine, I suggest using single-line comments for broader consistency. In general we only use block comments for explaining large paragraphs of text. Note also how the comment is written to fill up as much of the line as possible--prefer this versus breaking up multiple lines much earlier than the column limit.

Reply:
Okay, I will keep this in mind the next time. Now I have updated the comment will single-line comments. PTAL